### DISCUSSION TITLE
It doesn't work on win 11. "Undefined control sequence. \end{markdown}" and "File ended while scanning use of ^^M."

### DISCUSSION THREAD

**Discussion by zousiyu1995** [Witiko/markdown#171] [Q&A]
It works on my win 10 pc. **But It doesn't work on win 11**. I don't know why.

```
8: Undefined control sequence. \end{markdown}
8: Undefined control sequence. \end{markdown}
8: Undefined control sequence. \end{markdown}
8: Undefined control sequence. \end{markdown}
8: Undefined control sequence. \end{markdown}
8: Undefined control sequence. \end{markdown}
8: Undefined control sequence. \end{markdown}
8: Undefined control sequence. \end{markdown}
8: Undefined control sequence. \end{markdown}
8: Undefined control sequence. \end{markdown}
: File ended while scanning use of ^^M.
```

![image](https://user-images.githubusercontent.com/7608449/185619661-2531d1a0-cafa-4b68-acab-a5166799974c.png)

![image](https://user-images.githubusercontent.com/7608449/185784766-37deda59-381a-4f86-aef2-6ece1feb13e1.png)

**Comment by Witiko** [comment]
Seems related to a bug in the [lt3luabridge package][1] that I fixed previous week, see https://github.com/Witiko/lt3luabridge/pull/15. Can you please check the version of the lt3luabridge package that you have installed? Affected version of the lt3luabridge package is 2.0.0 2022-08-08, please use the TeX Live Manager (or similar if you are using a different TeX distribution) to update lt3luabridge to version 2.0.1 2022-08-16.

 [1]: https://ctan.org/pkg/lt3luabridge

**Comment by zousiyu1995** [comment]
> Seems related to a bug in the [lt3luabridge package](https://ctan.org/pkg/lt3luabridge) that I fixed previous week, see [Witiko/lt3luabridge#15](https://github.com/Witiko/lt3luabridge/pull/15). Can you please check the version of the lt3luabridge package that you have installed? Affected version of the lt3luabridge package is 2.0.0 2022-08-08, please use the TeX Live Manager (or similar if you are using a different TeX distribution) to update lt3luabridge to version 2.0.1 2022-08-16.

I am using MiKTeX, I cann't update this package to version 2.0.1.

**Comment by Witiko** [comment]
@zousiyu1995 I can see that MikTeX has not yet updated the `lt3luabridge` package to version 2.0.1: https://miktex.org/packages/lt3luabridge.

You can work around this bug by writing either `\def\markdownMode{0}` if you are using pdfTeX/XeTeX (if you don't know, that is likely the case) or `\def\markdownMode{2}`  if you are using Lua(Meta)TeX before `\usepackage{markdown}`. This will cause the Markdown package to not use the `lt3luabridge` package:

``` tex
\documentclass{article}
\def\markdownMode{0}
\usepackage{markdown}

\begin{document}

\begin{markdown}
hello **world**
\end{markdown}

\end{markdown}
```

The `\markdownMode` command will no longer be recognized in [Markdown 3.0.0][1], but that is not due to release in another year or so. However, having the `\markdownMode` command in your document should not hurt, Markdown will just ignore it from a certain point in the future.

 [1]: https://github.com/Witiko/markdown/discussions?discussions_q=label%3Amarkdown3
